src/build/mod.rs

```diff
@@ -234,35 +230,36 @@ async fn process_dockerfile<R: AsyncRead + std::marker::Unpin>(
         data_plane_run_script = format!("{data_plane_run_script} {port}");
     }
 
-    let bootstrap_script_content = r#"ifconfig lo 127.0.0.1\n echo \"enclave.local\" > /etc/hostname \n echo \"127.0.0.1 enclave.local\" >> /etc/hosts \n hostname -F /etc/hostname \necho \"Booting enclave...\"\nexec runsvdir /etc/service"#;
-
-    let installer_bundle_url = format!(
-        "https://cage-build-assets.{}/installer/{}.tar.gz",
-        ev_domain, installer_version
-    );
-    let installer_bundle = "runtime-dependencies.tar.gz";
-    let installer_destination = format!("{INSTALLER_DIRECTORY}/{installer_bundle}");
-
-    let egress = build_config.clone().egress;
-    let egress_settings = if egress.is_enabled() {
-        json!({
-            "ports": &egress.clone().get_ports(),
-            "allow_list": &egress.clone().get_destinations()
-        })
-    } else {
-        json!({})
-    };
-
     let mut dataplane_info = json!({
         "api_key_auth":  &build_config.api_key_auth(),
         "trx_logging_enabled": &build_config.trx_logging_enabled(),
         "forward_proxy_protocol": build_config.forward_proxy_protocol(),
         "trusted_headers": build_config.trusted_headers(),
     });
 
-    if egress.enabled {
-        dataplane_info["egress"] = egress_settings;
-    }
+    let egress = build_config.clone().egress;
+    let egress_config = if egress.is_enabled() {
+        dataplane_info["egress"] = json!({
+            "allow_list": &egress.clone().get_destinations()
+        });
+        r#"iptables -A OUTPUT -t nat -p tcp --dport 443 ! -d 127.0.0.1  -j DNAT --to-destination 127.0.0.1:4444\nip route add default via 127.0.0.1 dev lo\niptables -t nat -A POSTROUTING -o lo -s 0.0.0.0 -j SNAT --to-source 127.0.0.1\n"#
+    } else {
+        ""
+    };
+
+    let loopback_config = r#"ifconfig lo 127.0.0.1\n echo \"enclave.local\" > /etc/hostname \n echo \"127.0.0.1 enclave.local\" >> /etc/hosts \n hostname -F /etc/hostname \n"#;
```

Comment:
It might be cleaner to make these constants at the top of the file to clean up the code a bit? Wouldn't need all of the linebreaks which clutter the script.

Reply:
Just seeing this now, will add in follow up